apollo-ios/Sources/Apollo/Caching/ApolloStore.swift

```diff
@@ -173,9 +190,21 @@ public final class ApolloStore: Sendable {
 
   // MARK: -
 
+  /// A read-only transaction used to access an ``ApolloStore``'s cache data.
+  ///
+  /// A ``ReadTransaction`` is provided as a parameter of the transaction block of
+  /// ``ApolloStore/withinReadTransaction(_:)``. While inside of a read-only transaction block, concurrent write access
+  /// to the cache is blocked.
+  ///
+  /// - Note: A ``ReadTransaction`` should only be accessed from within the body of the
```

Comment:
Does Xcode, or the documentation viewer, display `Note` vs. `Important` differently? I noticed we had `Important` used elsewhere in this PR and wondered if there is a subtle difference?

Reply:
Yes, note displays in a grey box, important in a yellow box, warning in a red box.